server/@types/launchpad/index.d.ts

```diff
@@ -78,4 +79,6 @@ export type Establishment = {
   name: string
   displayName: string
   youth: boolean
+  contentHubURL: string
```

Comment:
Only thing is naming, it's called `prisonerContentHubURL` above and also the `uniLInkUrl` is called `selfServiceURL` elsewhere. Would be better to be consistent with one or the other?

Reply:
I have pushed changes to resolve this. Thanks.